### ISSUE TITLE
[Arcs] Self-hit that cannot be assigned (Judge, Truce)

### ISSUE THREAD

**Issue by turmoilbyrd** [LederCards/cards#82]
Discussion https://discord.com/channels/476234833572397056/963112189802201108/1308084671132729364

Assume:
- Judge is a regent
- [Judge's Chosen](https://cards.ledergames.com/card/ARCS-F2404) is a regent
- There is some imperial ships watching

In a battle, JC chooses to battle with Judge's ships, and rolled self-hit.

The way [Truce Law](https://arcs.seiyria.com/#14.3.1-truce-law) is written:

> Regents cannot harm Imperial ships except for hits they take as attacking ships in battle...

implies self-hit as attacker is also harm

So there can be a situation where the JC needs to assign self-hit but cannot due to Truce law. How to resolve this situation?

**Comment by turmoilbyrd** [comment]
Related, also deals with definition of harm #79

**Comment by turmoilbyrd** [comment]
Possible solutions:

- They cannot start this battle at all (the battle cannot happen because it may harm Judge's ships?)
- <s>They just ignore the self hits</s> (this is probably wrong)
- (IMO most plausible) Redefine self-hit as not harming. Because thematically self-hits are hits done by the defender. So it's not the responsibility of the Judge's Chosen

**Comment by jdyearsley** [comment]
Yes, self-hits should not be defined as harm for the reasons you stated.

**Comment by turmoilbyrd** [comment]
> Yes, self-hits should not be defined as harm for the reasons you stated. I'll add some errata to the campaign rulebook.

Also intercept, obviously
